assets/custom.css

```diff
@@ -2,6 +2,9 @@
     --border-color: #0f477e;
     --links-color: #10508E;
     --circle: 150px;
+    --background-color: #e8f1fb;
```

Comment:
The colour codes are in both capital and non capital letters. The same format should be used for all of them.